nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -529,7 +529,7 @@ in
     boot.initrd.luks.devices = mkVMOverride {};
 
     # Don't run ntpd in the guest.  It should get the correct time from KVM.
-    services.timesyncd.enable = false;
+    services.timesyncd.enable = lib.mkForce false;
```

Comment:
I am only aware of the default value that the module sets. Who else also sets this?

Reply:
@Mic92 I'm setting this in my own configuration.nix.

but i see that this is already enabled when you are not in container. I guess I'm enabling it for nothing since it is already true on my laptop.

Sorry for bothering, I will close this PR.

Reply:
Ah the user might. But do we really want a higher priority instead of a lower one? There valid use cases also to test timesyncd in a vm. `services.timesyncd.enable = lib.mkDefault false;`